src/FHIRConverterAPI/Program.cs

```diff
@@ -0,0 +1,117 @@
+using System.Net;
```

Comment:
`Program` isn't the most obvious entry point file name for me (though I'm not coming from a strong dotnet background, so maybe this is more common there?) - what about `Main`, `FhirConverterAPI` or `App`?